### PR TITLE
fix: removed tech channels

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #ISSUE-NO

### CODE REVIEW

File: docs/discord/discord-readme.md
Comment (LeaDevelop) [inline docs/discord/discord-readme.md:67]: I belive correct url in this line should be: `https://github.com/EddieHubCommunity/support/blob/main/docs/discord/discord-readme.md`

**Comment by LeaDevelop** [comment]
Looking at the entire file, from what I can tell there are some discord channels that are are missing in list or they to much. Which leads me to wonder, if they get unlocked at certain level for new members?

**Comment by eddiejaoude** [comment]
> Looking at the entire file, from what I can tell there are some discord channels that are are missing in list or they to much. Which leads me to wonder, if they get unlocked at certain level for new members?

Thanks Lea! It could be they got out of sync, could you let me know which ones you noticed are different?

File: docs/discord/discord-readme.md
Comment (LeaDevelop) [inline docs/discord/discord-readme.md:67]: Removed tech channels attempt looks legit to me, would approve, unless you'd like to make further changes and expand the scope of the PR itself?

**Review by LeaDevelop — COMMENTED** [review]
Removed tech channels attempt looks legit to me, would approve, unless you'd like to make further changes and expand the scope of the PR itself?

**Comment by LeaDevelop** [comment]
> > Looking at the entire file, from what I can tell there are some discord channels that are are missing in list or they to much. Which leads me to wonder, if they get unlocked at certain level for new members?
> 
> Thanks Lea! It could be they got out of sync, could you let me know which ones you noticed are different?

Sure thing, on it now

**Comment by eddiejaoude** [comment]
@LeaDevelop if you prefer, maybe I merge this PR and you can raise a new PR? Let me know what you prefer

File: docs/discord/discord-readme.md
Comment (eddiejaoude) [reply docs/discord/discord-readme.md:67]: I like to keep PRs small if possible, so happy to merge and further changes can come in another PR

**Comment by LeaDevelop** [comment]
> > Looking at the entire file, from what I can tell there are some discord channels that are are missing in list or they to much. Which leads me to wonder, if they get unlocked at certain level for new members?
> 
> Thanks Lea! It could be they got out of sync, could you let me know which ones you noticed are different?

This is what I currently see with my entry level without touching the Browse Channels option in the discord (I used that now and can see them all). Screenshot took before I touched Browse Channels option
eddihub-discord

My discord level according to Becca Lyria bot is 3.
I would propose that you list all channels and their descriptions members are able to reach and also mention to take a look at Browse Channel to see all other channels.
If you lock some channels after certain level of activity, would also list and explain what action has to be taken to reach it :)


Summary:
- would add disclaimer to use Browse Channels feature to find full channel list. 
    - perhaps natural flow for others. I use discord for +8 years and still missed it haha
- would list all channels, so people know what they can find and if they don't see it, they'll eventually look around and find the Browse Channels which empowers summary point one
- consider helper role in future, or add disclaimer high chance you'll get tagged if you use certain roles. I joined community and read through initial code of conduct and had idea that it'll be vibrant interactive journey but not sure if all perceive the entry into your community like that (Most common in my surroundings when I talk to people, they join discord servers and before anything else is done, they mute server for any pinging and that's understanding when you are in 10 ore more servers, discord goes brrrrrr 😅)

GENERAL DISCUSSION
#readme - ⚠️ this one is not listed and explained currently
#roles - ⚠️it only enlists if one enters it via readme channel and then it vanishes. BTW I didn't realize that if I pick certain role in the discord channel, members will assume I am expert in that area. Have you considered adding a helper role, if people want to be tagged for topics, although that may not really solve the problem, cos you'd have to separate interested into topic and able and willing to help on specific topic :) long separate topic, no need to address here.
#general  ✔️ 
#help  ✔️ 
#content-share - ⚠️ this one I see after using Browse Channels feature
#career - ⚠️ this one is currently not listed and explained
#bot-chat - ⚠️ this one I see after using Browse Channels feature

DAILY
#standup ✔️ 
#wellness ✔️ 
#achievements - ⚠️ this one I see after using Browse Channels feature

COMMUNITY DISCUSSION
#ideas ✔️ 
#first-timers ✔️ 
#public-speaking ✔️ 
#maintainers - ⚠️ this one is not explained in the file
#hackathons - ⚠️ this one i don't see, maybe cos of my entry level

RadioEddieStage 00/99 - ⚠️ Maybe also add description here, is this for hangout when you got events going on? Which also leads me to question do you consider having create channels on demands for members, when they help each other, or does all run through pull requests and text chat? Sometimes it's easier to convey via screen share and being able to explain show, explain and discuss problem via voice.

Locked channels
#TeamEddie
#Maintainers
#Sponsors

**Comment by LeaDevelop** [comment]
> @LeaDevelop if you prefer, maybe I merge this PR and you can raise a new PR? Let me know what you prefer

I think that'd be a best flow yeah :)

**Review by LeaDevelop — APPROVED** [review]

**Comment by eddiejaoude** [comment]
That is very well written Lea! 👍 thank you!

> > I set me reminder and will create issue and PR in one of the upcoming three days, heading off for today :)

Thank you! I have created an issue https://github.com/EddieHubCommunity/support/issues/8671, please comment in the issue and then GitHub will let me assign it to you

**Pull request merged**
